server/simple/Dockerfile

```
@@ -1,6 +1,6 @@
 # Official Dart image: https://hub.docker.com/_/dart
 # Specify the Dart SDK base image version using dart:<version> (ex: dart:2.12)
-FROM dart:stable AS build
+FROM dart:beta AS build
```

Can this go back to `stable`?

```suggestion
FROM dart:stable AS build
```

---

👍 good catch!

---

Actually, I'm sorry. I guess the docker images haven't been updated yet? I thought https://github.com/dart-lang/dart-docker/commit/a6701292b23f948e0c5ddb185d0d18812cf5501f would update it but maybe it's a manual process.

---

It's fine, it will be updated soon!